### PR TITLE
WORK:5249248:- Point2Site and SKU/Resize Gateway support

### PR DESCRIPTION
**Pull request by Nilambari**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nilambari__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilamd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-network/2015-06-15/swagger/network.json [hunk 1]
```diff
@@ -6374,6 +6430,82 @@
...
+      "properties": {
+        "ProcessorArchitecture": {
+          "type": "string",
+          "description": "VPN client Processor Architecture -Amd64/X86",
+          "enum": [
+            "Amd64",
+            "X86"
+          ],
+          "x-ms-enum": {
+            "name": "Architecture",
```
Comment (DeepakRajendranMsft) [inline arm-network/2015-06-15/swagger/network.json]: ProcessorArchitecture is a better name

**Comment by DeepakRajendranMsft** [comment]
LGTM

File: arm-network/2015-06-15/swagger/network.json (hunk 1 above)
Comment (Nilambari) [inline arm-network/2015-06-15/swagger/network.json]: Done.

**Comment by Nilambari** [comment]
Addressed the review comments. Can you Please help with the merge? Thanks!

**Pull request merged**